app/controllers/api/base_controller/parameters.rb

```diff
@@ -100,6 +100,14 @@ def sort_directive(klass, attr, order, options)
         end
         arel
       end
+
+      def determine_include_for_find(klass)
+        relations = @req.derived_include_for_find.select do |relation|
+                      klass.try(:reflect_on_association, relation)
```

Comment:
Is the try a nil check or a respond_to check?  If the former should we bail early from the method? `return nil if klass.nil?`

Reply:
I believe more of the latter, but if I am being honest, it is mostly a straight rip from this line:

https://github.com/ManageIQ/manageiq-api/blob/master/app/controllers/api/base_controller/renderer.rb#L345

Reply:
If we want, we could do a:

```ruby
def determine_include_for_find(klass)
   return nil unless klass.respond_to?(:reflect_on_association)
   # ...
end
```

And then drop the `.try`.

Reply:
make sense to me